modules/text2vec-openai/clients/meta.go

```diff
@@ -14,6 +14,6 @@ package clients
 func (v *vectorizer) MetaInfo() (map[string]interface{}, error) {
 	return map[string]interface{}{
 		"name":              "OpenAI Module",
-		"documentationHref": "https://beta.openai.com/docs/guides/embeddings/what-are-embeddings",
+		"documentationHref": "https://platform.openai.com/docs/guides/embeddings/what-are-embeddings",
```

Comment:
Can you also check `generative-openai` and `qna-openai` modules if they need also the same update?

Reply:
Good idea, added another commit.